product_landing_page/index.css

```diff
@@ -146,6 +146,58 @@ body {
   height: 315px;
 }
 
+.pricing-section {
+  margin: 4rem auto 0 auto;
+  display: flex;
+  justify-content: center;
```

Comment:
Jest problem z responsywnością w pewnej szerokości ekranu:
![rwd](https://user-images.githubusercontent.com/23278550/50536364-ee513000-0b53-11e9-901b-5f256de97bfc.png)
Całość działa naprawdę dobrze przy dużej szerokości ekranu i przy szerokości < 600px. Jedyną rzeczą do poprawy jest szerokość, przy której powinny pojawić się 2 boxy.

Tutaj do dodania width 100% i `flex-wrap: wrap`. Obrazowo mówiąc, działa tak: załóżmy, że masz 3 boxy i każdy z nich ma po 300px szerokości. Ekran masz szerokości 900px, jest ok, wszystko się mieści. Potem zmniejszasz ekran do 600px. Defaultowo flexbox będzie się starał "upychać" te elementy, żeby je zmieścić w mniejszym ekranie. Dając wartość `wrap`, eliminujemy to zachowanie. Jeżeli nie będzie na jakiś element wystarczająco dużo miejsca, flexbox wywali go do następnej linii. I o to nam tu chodzi :)